code/__DEFINES/reactions.dm

```diff
@@ -17,7 +17,7 @@
 #define NITRYL_FORMATION_ENERGY				100000
 #define TRITIUM_BURN_OXY_FACTOR				100
 #define TRITIUM_BURN_TRIT_FACTOR			10
-#define TRITIUM_BURN_RADIOACTIVITY_FACTOR	50000 	//! The neutrons gotta go somewhere. Completely arbitrary number.
+#define TRITIUM_BURN_RADIOACTIVITY_FACTOR	75000 	//! The neutrons gotta go somewhere. Completely arbitrary number. //NSV13
```

Comment:
```suggestion
#define TRITIUM_BURN_RADIOACTIVITY_FACTOR	75000 	//! The neutrons gotta go somewhere. Completely arbitrary number. //NSV13 increased this
```
Noting down what you changed is helpful in case upstream also changes it in the future.